src/schemas/makeReqSchema.ts

```diff
@@ -3,7 +3,7 @@ import { z } from "zod";
 export const MakeRequestSchema = z.object({
     title: z.string().min(4),
     description: z.string().min(10),
-    media: z.string().url(),
+    media: z.array(z.string()).min(1),
```

Comment:
The min(1) is for minimum number of images is it? I was thinking we should keep images optional, if thats what it was for.

Reply:
Yep, sure we can make that change. The min 1 was purely for form validation testing because the field is a custom field I made and not something that comes out of the box with react hook form